### PR TITLE
add package.json

### LINKED ISSUE #31: Add a package.json
Make sure it's marked as `private`. This should be a field in the package.json.  

Learn about `npm init --yes`. Put a link here.

### PR DESCRIPTION
**Pull request by VernL**
This pull request will close #31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RichardLitt/the-travel-shelf.git"
+  },
+  "keywords": ["independent", "bookstores", "index"],
+  "author": "Richard Littauer",
+  "license": "MIT",
+  "contributors": ["Joanna Katsifaraki", "Jose Esplana", "Hanie Jafari, Vernon Lillies" ],
```
Comment (RichardLitt) [inline package.json]: I believe these need to include more information. Take a look at some other package.json's that have it. You can find them through GitHub search. Same for the `author` field.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "bugs": {
+    "url": "https://github.com/RichardLitt/the-travel-shelf/issues"
+  },
+  "homepage": "https://github.com/RichardLitt/the-travel-shelf#readme",
```
Comment (RichardLitt) [inline package.json]: No need for `#readme`. 

Learn how to remove this and amend the commit. It's ok to force push to a branch (but _only_ to a branch_).

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "the-travel-shelf",
+  "version": "1.0.0",
+  "description": "A magazine and index of beautiful global independent bookstores",
+  "main": "index.js",
+  "directories": {
```
Comment (RichardLitt) [inline package.json]: Probably not needed. What does this do?

**Comment by RichardLitt** [comment]
@VernL Can you amend the commit and the pull request to be lowercase? Read this for how to write great commits: https://gist.github.com/stephenparish/9941e89d80e2bc58a153.

**Comment by Jojokats** [comment]
I'm having issues with pushing the changes. I changed the name from app to Base but after doing a git status, it's showing me in red that everything is deleted. 
If you can have a look and let me know what to do. Thanks

git-m

**Comment by RichardLitt** [comment]
@Jojokats Wrong issue, I think?

**Comment by Jojokats** [comment]
>@Jojokats Wrong issue, I think?

I don't understand....

**Comment by RichardLitt** [comment]
@Jojokats I think you posted this comment in the wrong PR. It's not relevant to #35.

**Comment by Jojokats** [comment]
oh no

**Comment by VernL** [comment]
I broke it :D

**Comment by VernL** [comment]
@RichardLitt I ran into a merge conflict. I now have two redundant commits. Is there an easy way get rid of them?

**Comment by RichardLitt** [comment]
Time to learn about squashing!

How did you get a merge conflict? I'm curious. You shouldn't have switched branches.

**Comment by RichardLitt** [comment]
Another option is to reset the branch to the master HEAD while keeping the files in the index, commit on top again, and then force push to this branch with the new commit only.

**Comment by VernL** [comment]
I'm curious too :D I was dual wielding command line interface and GitHub Desktop.
I modified  the package.json, ran git commit --amend. Everything going well. Then I switched over to GitHub Desktop to push the changes, but was out of sync. I fetched origin and pulled the changes and all hell broke loose. 

Noted about the branch change.  Should I kill this branch and submit a new pull request?

In the meantime, on to squashing.

**Comment by VernL** [comment]
It worked 👍

**Comment by RichardLitt** [comment]
> I fetched origin and pulled the changes and all hell broke loose.

You might have fetched origin master, by accident. That would mess things up.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RichardLitt/the-travel-shelf.git"
+  },
+  "keywords": ["independent", "bookstores", "index"],
+
+  "author": "Richard Littauer <richard.littauer@gmail.com> (https://burntfen.com)",
+  "license": "MIT",
+  "contributors": ["Joanna Katsifaraki", "Jose Esplana <jmesplana@gmail.com> (www.jey-em.com)", "Hanie Jafari me@hjafari.com (www.hjafari.com)", "Vernon Lillies <vernon.lillies@gmail.com>" ],
```
Comment (RichardLitt) [inline package.json]: Hanie needs `<>` around her email. 

Can you look for a `package.json` linter for me?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  "bugs": {
+    "url": "https://github.com/RichardLitt/the-travel-shelf/issues"
+  },
+  "homepage": "https://github.com/RichardLitt/the-travel-shelf",
+
```
Comment (RichardLitt) [inline package.json]: Doesn't look valid. It doesn't close. Where is the private tag?

**Comment by VernL** [comment]
> Can you look for a package.json linter for me?
online option : http://package-json-validator.com/ 
npm : https://www.npmjs.com/package/package-json-validator

**Comment by VernL** [comment]
Should be fixed.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "the-travel-shelf",
+  "version": "1.0.0",
+  "description": "> A magazine and index of beautiful global independent bookstores",
```
Comment (RichardLitt) [inline package.json]: No `> `.

Comment (VernL) [reply package.json]: check

**Comment by RichardLitt** [comment]
Well done!

**Pull request merged**